src/components/SearchResults/RequestSearchForm.js

```diff
@@ -25,17 +29,34 @@ export const RequestSearchForm = ({ closeForm, queryTerm = '' }) => {
           pediatric_cancer: '',
           approach: '',
           email: '',
-          email_updates: false
+          email_updates: false,
+          query: queryTerm,
+          request_type: 'search'
         }}
         validationSchema={RequestDataFormSchema}
         validateOnChange={false}
         onSubmit={async (values, { setSubmitting }) => {
-          // TEMP
-          await new Promise((resolve) => {
-            setTimeout(() => resolve(values), 1000)
+          const response = await requestData({
+            requestValues: {
+              ...values
+            }
           })
-          // eslint-disable-next-line no-console
-          console.log(values)
+
+          // redirects to the homepage after submission
+          const isSuccess = response.status === 200
+          push(
+            {
+              pathname,
+              query: {
+                message: isSuccess
+                  ? 'Request for Experiment Received!'
+                  : 'There was a problem with requesting the experiment. Please try again later',
+                status: isSuccess ? 'success' : 'error'
+              }
+            },
+            pathname
+          )
+
           setSubmitting(false)
         }}
```

Comment:
Let's pull this out of the jsx. I think it would be better to use two objects for each state and then switch between them, instead of switching both attributes.

Reply:
See the commit at 8f7051a5e4c5a69d09f1535edafc0ca36dab8e15.